WorkingInR/3_Importing_Data_in_R.Rmd

```diff
@@ -14,15 +14,50 @@ library("tidyverse")
 
 data(penguins, package = 'palmerpenguins')
 
-source("./Templates/biostats_theme.R")
+source("../Templates/biostats_theme.R")
 theme_set(theme_biostats)
 ```
 
-# Importing data
+### Before we start {- .facta .toc-ignore}
 
+You must be familiar with objects and data types, with using functions and arguments.
 
+# Foreword
 
+In this chapter, you will learn to import data from file into an R object. The R base package and several packages come with a set of functions that can do the job. Here, we will focus on the functions which are part of the package `tidyverse`. So, if it is not done yet, install the package with `install.packages("tidyverse")` and/or load it with `library(tidyverse)`.
 
+# Five functions to import data from a file
+
+Here, you will learn about 5 functions: `read_csv()`, `read_csv2()`, `read_excel()`, `read_table()` and `read_fwd()`. All these functions have the same purpose, namely to:
```

Comment:
I would just teach read_delim() (can cope with csv type files and I think everything that read_table can do) and read_excel. And then say that R can import many other types of files. fwf are rather rare in modern data